src/padding.jl

```diff
@@ -42,12 +41,67 @@ julia> pad_constant(r, (1, 2, 3, 4), 8)
  8  8  8  2  4  8  8  8  8
  8  8  8  8  8  8  8  8  8
  8  8  8  8  8  8  8  8  8
+
+julia> pad_constant(r, 1, 8)
+5×9 Matrix{Int64}:
+ 8  8  8  8
+ 8  1  3  8
+ 8  2  4  8
+ 8  8  8  8
 ```
```

Comment:
these ticks are in the wrong position. Also could reduce the number of examples, if the explanation of the behavior is clear